tests/test_async_client.py

```diff
@@ -9,9 +9,12 @@
 import lightkube
 from lightkube.config.kubeconfig import KubeConfig
 from lightkube.resources.core_v1 import Pod, Node, Binding
+from lightkube.generic_resource import create_global_resource
 from lightkube.models.meta_v1 import ObjectMeta
 from lightkube import types
 
+from .test_client import make_wait_custom, make_wait_deleted, make_wait_failed, make_wait_success, make_watch_list
```

Comment:
Added `tests/__init__.py` so that we can just import as `from .test_client` instead of `from test.test_client` and needing to add a custom path.

Reply:
ok